Snippets/Snippets_6/UpgradeGuides/5to6/CustomErrorHandlingBehaviour.cs

```diff
@@ -0,0 +1,59 @@
+namespace Snippets6.UpgradeGuides._5to6
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.Pipeline;
+
+    #region 5to6-customerrorhandlingbehaviour
+    class CustomErrorHandlingBehaviour : Behavior<ITransportReceiveContext>
+    {
+        public override async Task Invoke(ITransportReceiveContext context, Func<Task> next)
+        {
+            try
+            {
+                await next().ConfigureAwait(false);
+            }
+            catch (MessageDeserializationException)
```

Comment:
@mikeminutillo - The IManageMessageFailures had a method that'd get invoked on Serialization failures. This exception is there just to show that the code from that function needs to go in that catch block. 
Need to add something there or when the snippets get broken apart - show this as how to handle serialization failures in v6